tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -800,6 +800,11 @@ public function dataFileAsserts(): iterable
 			yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-6859.php');
 		}
 
+		if (PHP_VERSION_ID >= 80100) {
+			require_once __DIR__ . '/data/value-of-enum.php';
```

Comment:
Require is not needed here, make sure to run `composer dump` locally first.

Reply:
I checked it locally.. the tests do not work without this require line.

even if I run `composer dump` beforehand

Reply:
ahh.. I need to `/my/php8.1 composer dump` and then it work